primitives/src/pcs/prelude.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 Espresso Systems (espressosys.com)
+// This file is part of the Jellyfish library.
+
+// You should have received a copy of the MIT License
+// along with the Jellyfish library. If not, see <https://mit-license.org/>.
+
+//! Prelude
+pub use crate::pcs::{
+    errors::PCSError,
+    multilinear_kzg::{
+        srs::{MultilinearProverParam, MultilinearUniversalParams, MultilinearVerifierParam},
+        util::{compute_qx_degree, get_batched_nv, merge_polynomials},
+        BatchProof, KZGMultilinearPCS, Proof,
+    },
+    structs::Commitment,
+    univariate_kzg::{
+        srs::{UnivariateProverParam, UnivariateUniversalParams, UnivariateVerifierParam},
+        KZGUnivariateOpening, KZGUnivariatePCS,
```

Comment:
Speaking of which, I find the naming slightly different from my preference (obviously it's a matter of preference, nothing major)

I find `UniveriateKZGxxx` easier to read than `KZGUniveriatexxx`